tests/integration/test_logtest/test_ruleset_refresh/test_cdb_labels.py

```diff
@@ -0,0 +1,99 @@
+# Copyright (C) 2015-2021, Wazuh Inc.
+# Created by Wazuh, Inc. <info@wazuh.com>.
+# This program is free software; you can redistribute it and/or modify it under the terms of GPLv2
+
+import json
+import os
+import shutil
+
+import pytest
+import yaml
+from wazuh_testing.tools import WAZUH_PATH
+from wazuh_testing.tools.configuration import set_section_wazuh_conf
+import wazuh_testing.tools.configuration as conf
```

Comment:
We should import only what is necessary instead of the entire module, as it has been done in the line above.